python/tvm/relax/backend/contrib/__init__.py

```diff
@@ -0,0 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Relax backends contrib"""
+from . import cublas
+from . import cutlass
```

Comment:
Does keeping an empty init file fix the issue? We don't want `from . import cutlass` (probably cublas as well) here.

Reply:
The empty init file is fine.

Reply:
Here is the summary of the previous issue for reference:

* It is fine for `backend/contrib/cutlass.py` to depend on `contrib/cutlass/build.py`
* But `contrib/cutlass/build.py` does `import relax`, which transitively imports `backend/contrib/cutlass.py`